pages/about.html

```diff
@@ -4,9 +4,49 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="./style.css"/>
+  <title>About Me</title>
 </head>
+
 <body>
+  <header>
+    <div class="head-title"> About Me</div> 
```

Comment:
I recommend using h1 here instead of a div, as the h1 tag has more semantic meaning.